Description: Disable the default handlers.
 Re-enable them if you want the previous behaviour.
 .
 drbd8 (2:8.3.13-2) unstable; urgency=low
 .
   * Comment-out default handlers for pri-on-incon-degr/pri-lost-after-sb in '/etc/drbd.d/global_common.conf'
   ** Re-enable them if you want to improve your availability in a DRBD cluster setup. (Closes: #576511)
Author: Philipp Hug <debian@hug.cx>
Bug-Debian: http://bugs.debian.org/576511
Origin: other

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- drbd8-8.3.13.orig/scripts/global_common.conf
+++ drbd8-8.3.13/scripts/global_common.conf
@@ -7,9 +7,12 @@ common {
 	protocol C;
 
 	handlers {
-		pri-on-incon-degr "/usr/lib/drbd/notify-pri-on-incon-degr.sh; /usr/lib/drbd/notify-emergency-reboot.sh; echo b > /proc/sysrq-trigger ; reboot -f";
-		pri-lost-after-sb "/usr/lib/drbd/notify-pri-lost-after-sb.sh; /usr/lib/drbd/notify-emergency-reboot.sh; echo b > /proc/sysrq-trigger ; reboot -f";
-		local-io-error "/usr/lib/drbd/notify-io-error.sh; /usr/lib/drbd/notify-emergency-shutdown.sh; echo o > /proc/sysrq-trigger ; halt -f";
+		# The following 3 handlers were disabled due to #576511.
+		# Please check the DRBD manual and enable them, if they make sense in your setup.
+		# pri-on-incon-degr "/usr/lib/drbd/notify-pri-on-incon-degr.sh; /usr/lib/drbd/notify-emergency-reboot.sh; echo b > /proc/sysrq-trigger ; reboot -f";
+		# pri-lost-after-sb "/usr/lib/drbd/notify-pri-lost-after-sb.sh; /usr/lib/drbd/notify-emergency-reboot.sh; echo b > /proc/sysrq-trigger ; reboot -f";
+		# local-io-error "/usr/lib/drbd/notify-io-error.sh; /usr/lib/drbd/notify-emergency-shutdown.sh; echo o > /proc/sysrq-trigger ; halt -f";
+
 		# fence-peer "/usr/lib/drbd/crm-fence-peer.sh";
 		# split-brain "/usr/lib/drbd/notify-split-brain.sh root";
 		# out-of-sync "/usr/lib/drbd/notify-out-of-sync.sh root";
